ansible/roles/events/templates/graphql-api.env

```diff
@@ -0,0 +1,15 @@
+port: {{ graphql_port }}
+debug: true
+NODE_DEBUG: http node index.js
+organization: ala
+apiEs: http:///es-api:{{ es_port | default('4001') }}
+apiEsKey: {{ es_api_key }}
+es2vt: http:///es2vt:{{ es2vt_port | default('4002') }}
+apiDownloads: "{{ downloads_url }}/dataset/{datasetKey}"
+datasetEml: "{{ collectory_url }}/ws/eml/{datasetKey}"
+alaNamematching: "{{ namesearch_url }}/api"
+ala:
+  namematching: "{{ namesearch_url }}/api"
+  bie: {{ bie_service_url }}
+  biocache: {{ biocache_ws_url }}
+  images: {{ media_store_url }}
```

Comment:
`media_store_url` is used in other services without the /ws/ so will be great to user a different name to avoid conflicts.